qiskit/quantum_info/synthesis/one_qubit_decompose.py

```diff
@@ -85,6 +87,10 @@ class OneQubitEulerDecomposer:
           - :math:`Z(\phi) Y(\theta) Z(\lambda)`
           - :math:`e^{i\gamma} U_1(\phi+\pi).R_X\left(\frac{\pi}{2}\right).`
             :math:`R_Z(\theta+\pi).S_X\left(\frac{\pi}{2}\right).U_1(\lambda)`
+        * - 'ZSXX'
+          - :math:`Z(\phi) Y(\theta) Z(\lambda)`
+          - :math:`e^{i\gamma} U_1(\phi+\pi).R_X\left(\frac{\pi}{2}\right).`
```

Comment:
Should here (and above), be in terms of `R_Z`?

Reply:
Yeah, it's just sloppy copy and paste that's continued to propogate. I'll update it

Reply:
Updated in: https://github.com/Qiskit/qiskit-terra/pull/5937/commits/5b0efff5fa5dd59293f4043240c8e03fd5f2ebd6